backend/backend/fixtures/initial_data.json

```diff
@@ -0,0 +1,161 @@
+[
+    {
+        "model": "backend.content",
+        "pk": 1,
```

Comment:
Do we need to use pk? Can cause problem if we add more content because the key is used